src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -51,6 +51,7 @@ class Analyzer : public glass::View {
   void PrepareGraphs();
   void RefreshInformation();
   void AbortDataPrep();
+  void LoadFeedforwardGains(bool combined = false);
```

Comment:
It's not loading anything, just displaying it.
```suggestion
  void DisplayFeedforwardGains(bool combined = false);
```

Reply:
Yeah that makes more sense